tests/kernel/wave/attention/paged_attention_test.py

```diff
@@ -204,12 +204,8 @@ def testPagedFlashDecoding(
         num_kv_heads = key_cache.shape[2]
         head_size_kv = value_cache.shape[3]
 
-    permuted_key_cache = key_cache.view(num_seqs, -1, num_kv_heads, head_size).permute(
-        [0, 2, 1, 3]
-    )
-    permuted_value_cache = value_cache.view(
-        num_seqs, -1, num_kv_heads, head_size_kv
-    ).permute([0, 2, 3, 1])
+    permuted_key_cache = key_cache.view(num_seqs, -1, num_kv_heads, head_size)
+    permuted_value_cache = value_cache.view(num_seqs, -1, num_kv_heads, head_size_kv)
```

Comment:
No longer permuted, so change the variable names?

Reply:
Will do thank you